File: challenges/codeeval/medium/027/victor.ayalamarin.py
__author__ = "VICTOR"
__date__ = "$Nov 26, 2015 12:25:19 PM$"

#file = open(sys.argv[1], 'r')
file = open('C:\\Users\\VICTOR\\Documents\\Aprueba.txt','r')
for line in file:
    linea=line.strip(' \n')
    if linea:
        print (str(bin(int(linea)))[2:])
file.close()

File: challenges/codeeval/easy/092/victor.ayalamarin.java
package codeeval;

import java.io.File;
import java.io.IOException;
import java.util.Scanner;
import java.util.Stack;
/**
 *
 * @author VICTOR
 */
public class PenultWord {
    public static void main(String[] args) throws IOException {
        String linea;
        String ruta="C:\\Users\\VICTOR\\Documents\\Aprueba.txt";
        File file = new File(ruta);
        //File file = new File(args[0]);
        try (Scanner leeArc = new Scanner(file)) {
           while (leeArc.hasNext()){
                linea=(leeArc.nextLine());
                Scanner leeLin = new Scanner(linea);
                Stack palabras =new Stack();
                do {
                    palabras.push(leeLin.next());
                }while(leeLin.hasNext());
                int k=palabras.size();
                palabras.remove(k-1);
                System.out.println(palabras.pop());
                }
            leeArc.close();  
        } 
    }
